### ISSUE TITLE
Wrong scroll position for results list

### ISSUE THREAD

**Issue by ltGuillaume** [TBog/TBLauncher#158]
# Description #

Since updating to v6.4-add_pages, I have seen that the results list has a scoll position issue.

## Steps to Reproduce ##

To reproduce the behavior:
1. Set the Home button to directly open the search desktop
2. Open search and scroll down, so the keyboard is hidden
3. Start an app
4. Press the Home button
5. The search desktop will appear, as well as the keyboard, but the scroll position isn't at the bottom, so when you start searching for an app, it won't show up until you scroll to the bottom.

## Expected behavior ##

When showing the keyboard in this situation, the scroll position needs to be reset. I probably see this behavior now due to the fact that I previously saw a glance of the widget desktop before the app was started (due to "Widget mode after launch"). I like the new behavior (without that glance of the widget mode first), but since going back to widget mode before searching again does not show this behavior, I figure they're related.

## Device info ##

  * OS: Android 10
  * TinyBit Launcher version: v6.4-add_pages

**Comment by ltGuillaume** [comment]
This happens a lot, any idea what causes it? Can I get you more specific info?

**Comment by TBog** [comment]
I have the same issue but had no time to work on it.

**Comment by ltGuillaume** [comment]
It seems that there were some changes regarding the scroll position since the latest version: Previously the scroll position was retained when switching from one label's results list to another (via quick list). This, of course, wasn't desirable either. Perhaps something similar is going on here?

**Comment by ltGuillaume** [comment]
Yeah I switched back to the (officially) v6.4 final, I can't work with this. I also had another problem with the latest test build. I'll create a new issue for it.

**Comment by TBog** [comment]
Should be fixed in v6.5
Can you give more feedback on the latest build? [v6.6-dev.2](https://github.com/TBog/TBLauncher/releases/tag/v6.6-dev.2)

**Comment by ltGuillaume** [comment]
You're right, I haven't seen this behavior for some time.
